models/DataObject/Classificationstore/CollectionConfig.php

```diff
@@ -25,13 +26,18 @@
 final class CollectionConfig extends Model\AbstractModel
 {
     /**
-     * Group id.
-     *
-     * @var int
+     * @var bool
+     */
+    public static $cacheEnabled = false;
+
+    /**
+     * @var int|null
```

Comment:
Also a candidate for the trait and additionally it could be `private`.

Reply:
Done...